lib/rules/no-else-return.js

```diff
@@ -33,7 +35,24 @@ module.exports = {
          * @returns {void}
          */
         function displayReport(node) {
-            context.report({ node, message: "Unnecessary 'else' after 'return'." });
+            context.report({
+                node,
+                message: "Unnecessary 'else' after 'return'.",
+                fix: fixer => {
+                    const sourceCode = context.getSourceCode();
+                    const startToken = sourceCode.getTokenByRangeStart(node.start);
+                    const elseToken = sourceCode.getTokenBefore(startToken);
+                    const source = sourceCode.getText(node);
+                    let fixedSource;
+
+                    if (startToken.type === "Punctuator") {
+                        fixedSource = source.substr(1, source.length - 2);
```

Comment:
I definitely appreciate the feedback. This type of semantic parsing type of code, and code modification is new to me, and I'm not familiar with all the ins and outs so I'm grateful for your detailed feedback. I'll give it another go later and update. Thanks again for your time!